src/System.Private.CoreLib/shared/System/Text/StringBuilder.cs

```diff
@@ -1663,65 +1661,36 @@ internal StringBuilder AppendFormatHelper(IFormatProvider provider, string forma
                         ch = format[pos];
                         pos++;
 
-                        // Is character a opening or closing brace?
-                        if (ch == '}' || ch == '{')
-                        {
-                            if (ch == '{')
-                            {
-                                // Yes, is next character also a opening brace, then treat as escaped. eg {{
-                                if (pos < len && format[pos] == '{')
-                                    pos++;
-                                else
-                                    // Error Argument Holes can not be nested.
-                                    FormatError();
-                            }
-                            else
-                            {
-                                // Yes, is next character also a closing brace, then treat as escaped. eg }}
-                                if (pos < len && format[pos] == '}')
-                                    pos++;
-                                else
-                                {
-                                    // No, then treat it as the closing brace of an Arg Hole.
-                                    pos--;
-                                    break;
-                                }
-                            }
-
-                            // Reaching here means the brace has been escaped
-                            // so we need to build up the format string in segments
-                            if (unescapedItemFormat == null)
-                            {
-                                unescapedItemFormat = new StringBuilder();
-                            }
-                            unescapedItemFormat.Append(format, startPos, pos - startPos - 1);
-                            startPos = pos;
-                        }
+                        if (ch == '}')
+                            // Argument hole closed
+                            break;
```

Comment:
nit: I know the existing code didn't have curly brackets, but since you're adding a new block it should have them to match our coding guidelines.

Reply:
Okay no problem, I was just trying to match the existing style in the rest of the method.